examples/sampling-adaptive-covariance-rao-blackwell.ipynb

```diff
@@ -0,0 +1,150 @@
rt8XlHv+Cyq+Oo8Kg7+nRKfh5Chbh8v7fueX93qy+M0unP7zd7IVLE69wZOo0XsEV08f5dNn27D5\np1mICCFtutBv0gKy5ynAiP5dmP35WJxOZ7zx3Fe2AiOn/UhUVCSdW9Zl/uyZCXa3+QcE8NJrI/h6\nwWqCsmShc7uW9Hq6m17VuBljphhjgo0xwfnS6+6jx36HItXBL/k3G6ekydGPEok/L/jNtzqUNCNZ\nSUZcuojIm+7XRUTkjllfGUmHzk9RsnQ5xr3/1h17srR/ojs5cuXhmwmj4/yjWqnmQ9z3YG1WTv+I\nC6cS/zYakDmILHnv5fKx+AfG/9m8kj+/+4gcZWpT/sWZFGn5AtlLVsMeeOcy6f5Zc5OzfF2KtRtC\nhVfmUrjlAKKvXeK3j19k6Ts9uXr6b4rXaUXTkXPJW7oKiz4ZztQ3XyTy5g3yF7ufPuPnUrlhG2Z9\n9iGvv/A0ETfC442rROnyjJu7itIVqzJ88PP06NyGg/v2JPh5K1Spxtyl6+nz4hCWLvieuiFVWf/b\nr4n+O6k07sYl+Hc3FLv7nTBTynlyMNPRmLa29ZSQ04kfoJI9JvMZUAuImZN7DcjQ0y78/PwYOWoM\n/xw7wndfTb7lvaAsWek7cCg7N//OpjXL7jhWRBj63sfY7H58+1ZfIsKvJXq+XMXLc/7gdpxOR5zv\n7/z5awLzFKJEx7fwy5zN489hD8xMvhqtKf/iDAq3eIEbZ46w/M0n2LvoCwJz5Kbuy59SsUN/jm9a\nzoT+nbh64SwBmTLT/tUxNH/uNfb+voIXn2rF+X/jX3YmT/57eHfq9zwz5F0O7tpGx6a1GNj/GY7/\nfedNozECM2Wi/+A3+WreCgICA2nfqgnjP/pQ96NJz45vBAwUr2N1JLeYGt2KaPzoZU/wLgnlltwk\nU9MY8zxwE8AYcwlIHdezFmrYpBl16jfm83GjuXDu7C3vdezSi2L3l+WLMW9yM/zO3ffuKVyM18ZO\n4fzxI3z3zgsJzkYDeKhFa26GXeD0n7/f8Z5xOrl+fDc5y9VFkrlistj9yFezLeX6TydH2TrsnvcZ\nK0Y9T9SN65Rr1ZOHXvzI1X3WvxOX/j2BiFCnQ0+6jJzCxVPHeLFzU/Zs3Rhv+zabjdZdejNlyUba\ndH2WdUsX0Lp+VZ7v3YX9e3bGe1ylB6szZ8lvNG7RlpHDh9Gze1ciIiKS9RlVKnf0d7AHQKHUdQPk\neXIw31GH9vbfyMlVq8NJ9ZKbZKJExI57+RYRyQfE3xmfgfzvo3HcvHmDT/936wwUf39/3vngE86e\nOsHcqZ/EeWyVkLr0e+tDDm/9nQUfvZHgt/RytRsRlOce9i/+Mt56ksQVmePinzU3xTu+QeGWAwj7\nayvL3unBzbCL3PtgPeoN+ZzI61f47MXOXPrXtXxcmZr16TP+ewIzZ+H1Zzowe/JHREfFv9Ngthy5\n6DnobaYu3Uzb7n0J/W01jzerQ4/Ordm6aX2cxwRlycqYz77ihSFvsXj+HB5r3YJr1xK/+lNpzKGV\nrlllqfDmx2mOFmSWSJ60r7Y6lFQvuUlmPDAfyC8i7wG/A6NSLKo0rGSp0jz9zLPMnz2TI4duXZat\nao3a1G/VgQUzP+fU8bin8TZ57Eme6DuIHSvn88vM8fGex+7nT6MufblweCf/7rz1j7HYbPjnyE/E\nxYRXSo7hjIok7K9QrhzcxM3z/9zxvoiQr0Zr7u86mohLp1gxsjcR166Qp2RF6r86BUfEDT4f2JWw\n866pzAWKl6LvxHlUqNuMWRPH0K9jY3ZsXJdg0sydrwA9Br7JtOVb6dJ/KH/t3cnTHZrxZLsm7Ai9\n84pIROj9wmDe/2Qq2zatp33r5ly7qt8q040Lf7lufizTwupI4nTIFGadoyJd/VaBI9rqcFK15M4u\n+xbX8v6jcK2I3NYYMzclA0vLBg4eRqbMQXz+8eg73ntj+Pv4+fkxa+KYeI/v/NwrVG3anjVff8q+\n9av+j73zDovi6uLwe3eX3gVUEFTsPfbee+/dxN6Nny12jcaWGGONRmOJvSXG3nuJDXtv2BULqCBK\nXfZ+f+yi4C6CWBZh3ufZh92ZO3fOLDBn7r3n/E687YrUaoatqweXNsw1uoHbZcjJq3sX3ntjl1Ly\n9Mgazk9qxs0lg7m1fBhXfm/PhRldCLltnDTpkKUwWVuPJ+LZA/ZOG4ROF41LppyUGzCTiJDnzB3U\nifDX+hu9tb0DzYdPpdWPvxP+OoQfuzanR9OqbF296L1qAfaOTjTv2pf520/QedBYHtz2o22janTv\n2Ar/B/eM2tdt3JKJsxZx/vQJWjVvrEydpRSuGUQ7ctYyrx3vYXl0VTzEc/CL/39UIenRZUullFel\nlLOklDOllFeEEEs/tderiUgAACAASURBVHFfK27u7nTq2oOdW9YZJR6mTe9B6w7dOLR9PQ/vmF7o\nFkIwfPwUMuTIz5qJAwkOMB3FotZYUK1tL57fumg0mslavDxRwU8JD4g/Wi3QdwMPd8xG45YVxyqD\ncKo9FrtibdFFvsJv0QACjhsnlzpkKYRX7e8J8TvB9e36X7lrlnyU6jmR4Ic3mTu0O9Gxnuzylq9J\nn4U7qPe/0WijIpkzfggdqhakU90y/DF2EPs2rzEZJGBlY6tfs9lyjNa9BnHy0B4aVi7GqkXGDrV6\nnYb89NsfHP9vP106tleCAVIC17ZBunzgkmCJKrOxR1eIAOkEpxeb25RkTVKny+LonxjWZ4p8vDkp\nh87dewHw70rjP8BvO/VCrdaw7e/4/zgtLK34ceo8orVRbJsd/0xk4RqNsXX14PLGeXG2e3xTDoDg\nq0dMHhcdGcbDPYuw8MiHY9XBWHoXxiJtdmzy1salwSQsvArzYPscIp49NDrWtUgdnHKV5uL6uYS+\n0Ac4eBQoQ9H2I3h62ZeFYwbFudFbWFpRon4b
```

Comment:
One of these notebooks should maybe mention the sudden convergence after the initial phase. But explain why we still don't just start off adapting

Reply:
Added some notes about this; good point.